### PR TITLE
[Gtk4Prep] Slots cleanup Gtk4prep

### PR DESCRIPTION
**Pull request by jeremypw**
Gtk4 Prep:
 - No issues found

Cleanup:
 - Merge single use code
 - Modernize construction
 - Fix whitespace and layout

### CODE REVIEW

File: src/View/Miller.vala [hunk 1]
```diff
@@ -47,10 +47,15 @@ namespace Files.View {
             }
         }
 
-        public Miller (GLib.File loc, View.ViewContainer ctab, ViewMode mode) {
-            this.ctab = ctab;
-            this.root_location = loc;
+        public Miller (GLib.File loc, View.ViewContainer _ctab, ViewMode _mode) {
```
Comment (danirabbit) [inline src/View/Miller.vala:50]: It looks like this takes an argument `_mode` that is never used?

**Review by danirabbit — COMMENTED** [review]
One comment/question otherwise looks good

File: src/View/Miller.vala (hunk 1 above)
Comment (jeremypw) [reply src/View/Miller.vala:50]: True - its always been like that I think.  I guess the idea was to allow the possibility of a multi-slot view with the individual slots showing grid or list views instead or column views.  In the event, that was never implemented and the slots are hard-coded as column views so I guess that parameter is  redundant.  I'll remove it.

**Review by danirabbit — APPROVED** [review]
:rocket:

**Pull request merged**